spec/cypress/integration/accessibility.js

```diff
@@ -4,7 +4,7 @@ describe("Accessibility", () => {
     cy.checkA11y();
   });
 
-  it("Login should be accessible", () => {
+  it.only("Login should be accessible", () => {
```

Comment:
I don't think we want to keep this change ;) Nor the comment on line 40.

Reply:
Line 40 was intentional - it'll be back when the preferred name stuff is done

Reply:
Ah, alright.